.doc_gen/metadata/firehose_metadata.yaml

```diff
@@ -0,0 +1,18 @@
+# zexi 0.4.0
+firehose_PutRecordBatch:
+  title: "&FHLong; batch record write"
```

Comment:
Standard format for title is more like:
```
Write multiple &FH; records using an &AWS SDK;
```
(also, don't use long names in titles)